test/dynamo/test_aot_autograd_cache.py

```diff
@@ -292,6 +292,58 @@ def fn(x, y):
         self.assertEqual(counters["aot_autograd"]["autograd_cache_hit"], 1)
         self.assertEqual(counters["aot_autograd"]["autograd_cache_saved"], 1)
 
+    @inductor_config.patch("fx_graph_remote_cache", False)
+    @inductor_config.patch("fx_graph_cache", True)
+    @functorch_config.patch({"enable_autograd_cache": True})
+    def test_vmap(self):
+        """
+        make
+        """
+
+        def fn(x, y):
+            f = lambda x, y: (x * y + 1).sum(dim=0)  # noqa: E731
+            vmapped = torch.vmap(f)(x, y)
+            return vmapped.sum(dim=0)
+
+        torch._functorch.config.strict_autograd_cache = True
```

Comment:
this needs to be in a context manager or something otherwise you will pollute global test state

Reply:
Yeah sorry i was debugging sth and forgot to delete